mumblepahelper.ui

```diff
@@ -564,6 +717,16 @@
     <string>&amp;Rescan</string>
    </property>
   </action>
+  <action name="action_SetUserPluginsLocation">
+   <property name="text">
+    <string>&amp;Set user plugins location...</string>
```

Comment:
This text differs from the text in the dialog. The dialog uses directory, but this uses location.
I think they should both read directory, and not location.